resources/views/community/components/developer/sets-page.blade.php

```diff
@@ -0,0 +1,153 @@
+@props([
+    'user' => null,
+    'consoles' => [],
+    'games' => [],
+    'sortOrder' => 'title',
+    'filterOptions' => [],
+    'userProgress' => null,
+])
+
+<x-app-layout
+    pageTitle="{{ $user->User }} - Developed Sets"
+    pageDescription="Games developed by {{ $user->User }}"
+>
+    <x-user.breadcrumbs :targetUsername="$user->User" currentPage="Developed Sets" />
+
+    <div class="mt-3 -mb-3 w-full flex gap-x-3">
+        {!! userAvatar($user->User, label: false, iconSize: 48, iconClass: 'rounded-sm') !!}
+        <h1 class="mt-[10px] w-full">{{ $user->User }}'s Developed Sets</h1>
```

Comment:
Not sure if it matters too much, but there is a subtle difference here from how the Completion Progress page renders its header.

**This page**
![Screenshot 2023-11-08 at 4 15 09 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/d22e55e4-c751-44a8-8f04-84a62705c22f)

**Completion Progress page**
![Screenshot 2023-11-08 at 4 15 16 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/a831d912-19a9-4af3-9ba3-f21b64991856)

On the Completion Progress page, if the username ends with an "s", an apostrophe is added without an additional "s" at the end.

Reply:
Google says both ways are correct, but seems to favor using the `'s` for singular proper names.

Merriam-Webster says the second form implies that there's more than one Jamiras (as in a family name): "The Jones' house" is a house belonging to the Jones family. "Jones's dog" is a dog belonging to a guy named Jones.

The Associated Press requires that "any plural noun, including names that end with S, should not include an additional S after the apostrophe", though I don't understand how a name ending in S is automatically a plural noun. Maybe this one is misleading?

I also found this:
> Major style guides like APA Publication Manual, MLA, and Chicago recommend using apostrophes and extra letters S. This rule applies to both proper and common nouns.
> Other formal writing style guides only recommend adding an apostrophe after singular nouns ending in S, specifically proper nouns. This structure is also more common in informal writing or a hip style of texting.

Reply:
Thanks for sharing; it's hard to argue against APA or MLA guidelines. I'll make a mental note to adjust this on the Completion Progress page later.